docs/pages/orbit.md

```diff
@@ -162,7 +162,7 @@ The bullets serve two purposes: they mark the current slide, and can be clicked
 ```
 
 <!-- <div class="callout"><p>
-  The new Orbit slider was designed to be a tool for rapid prototyping. While you are welcome to use it in production, if you want something more robust, we welcome you to try [Owl Carousel](http://owlgraphic.com/owlcarousel/).
+  The new Orbit slider was designed to be a tool for rapid prototyping. While you are welcome to use it in production, if you want something more robust, we welcome you to try [Owl Carousel](https://owlgraphic.com/owlcarousel/).
```

Comment:
This domain is controlled by criminals now who redirect to malware websites. We should update it to https://owlcarousel2.github.io/OwlCarousel2/ or Swiper.

Reply:
Another good find.

Reply:
This was just commented out of the page anyways. I have removed it